mmv1/templates/terraform/examples/cloudfunctions2_secret_env.tf.erb

```diff
@@ -0,0 +1,67 @@
+# [START functions_v2_basic]
```

Comment:
```suggestion
```
Mind deleting the lines starting with `# [` (looks like there are 4 introduced in the PR) ? They serve a different purpose and are not needed in these examples

Reply:
Done